cli/api_utility_commands.go

```
@@ -0,0 +1,52 @@
+	Category: CategoryApi,
+}
+
+var dashboardTokenPrintCommand = &cli.Command{
+	Name:      "print",
+	Usage:     "Print your claim token",
+	ArgsUsage: "",
+	Action: func(ctx context.Context, cmd *cli.Command) error {
+		resp := callDashboardApi[UserResponse]("GET", "/web/user")
+		if resp.Type == "success" {
+			printSuccess("User request was successful")
+			printInfo(resp.Payload.Settings.ClaimToken)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to github.com/husarnet/husarnet/cli/v2.printInfo

(govet)